.gitignore

```diff
@@ -1,6 +1,15 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
 
+# OS
+.DS_Store
+
+# Editors
+.idea
+*.sublime-project
+*.sublime-workspace
+*.sublime-workspace
+
```

Comment:
It is their responsibility.

Otherwise, everyone who has some particular file that needs to be ignored will make a pull request to have it added the `.gitignore` of the project?